tools/generator/src/Generator+AddTargets.swift

```diff
@@ -146,7 +146,7 @@ Product for target "\(id)" not found in `products`
             return nil
         }
 
-        let publicHeaders = inputs.hdrs
+        let publicHeaders = inputs.hdrs.filter(\.path.isHeader)
```

Comment:
I don't think this is needed? Only the `inputs.nonArcSrcs` and `inputs.srcs` filters.